File: xiajiwangluo_store/src/main/webapp/js/service/SublistService.js
//服务层
app.service('SublistService',function($http){

    //查询子账号信息
    this.findAll1=function(count){
        return $http.get('../sublist/findAll.do?count='+count);
    };


    //修改子账号信息
    this.update=function(shopSublist){
        return $http.post('../sublist/update.do',shopSublist);
    };
});

File: xiajiwangluo_store/src/main/java/com/xiajiwangluo/service/SublistService.java
package com.xiajiwangluo.service;

import com.xiajiwangluo.bean.ShopSublist;
import com.xiajiwangluo.bean.entity.Result;

import java.util.List;


public interface SublistService {

    ShopSublist findAll(String sid,Integer count);

    Result Update(ShopSublist shopSublist);
}
